src/reducers/pending-breakpoints.js

```diff
@@ -81,12 +81,16 @@ function addBreakpoint(state, action) {
     return state;
   }
 
-  // when the action completes, we can commit the breakpoint
-  const { value: { breakpoint } } = action;
-  const locationId = makePendingLocationId(breakpoint.location);
-  const pendingBreakpoint = createPendingBreakpoint(breakpoint);
+  if (action.type === "ADD_BREAKPOINT") {
```

Comment:
Not sure about this change yet. Somewhat related to the remaining error. Just involves some type refinements in certain places.

Reply:
I noticed this function is called for both add and enable breakpoints but I don't see the `value` property on the enable breakpoint action @jasonLaster Could you help with a little clarification?